recipe/meta.yaml

```diff
@@ -11,29 +11,34 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   entry_points:
     - black = black:main
```

Comment:
https://github.com/psf/black/blob/f6c139c5215ce04fd3e73a900f1372942d58eca0/setup.py#L132

I think it should be `- black = black:patched_main`